build/mtm1m3ts_sim/Dockerfile

```diff
@@ -0,0 +1,88 @@
+ARG cycle
+ARG hub
+
+FROM ${hub}/deploy-env:${cycle}
+
+LABEL maintainer Tiago Ribeiro <tribeiro@lsst.org>
+
+USER root
+
+RUN dnf install -y dnf-plugins-core epel-release && \
+    dnf config-manager --set-enabled powertools && \
+    dnf -y --enablerepo=extras install make \
+    gcc-toolset-11 \
+    git \
+    catch-devel \
+    boost1.78 \
+    boost1.78-devel \
+    readline-devel \
+    snappy-devel \
+    spdlog-devel \
+    jansson \
+    iproute \
+    yaml-cpp-devel && \
+    dnf clean all
+
+RUN ln -s /usr/lib64/libboost_filesystem.so.1.78.0 /usr/lib64/libboost_filesystem.so && \
+    ln -s /usr/lib64/libboost_iostreams.so.1.78.0 /usr/lib64/libboost_iostreams.so && \
+    ln -s /usr/lib64/libboost_program_options.so.1.78.0 /usr/lib64/libboost_program_options.so && \
+    ln -s /usr/lib64/libboost_system.so.1.78.0 /usr/lib64/libboost_system.so && \
+    ln -s /usr/lib64/libcurl.so.4 /usr/lib64/libcurl.so && \
+    ln -s /usr/lib64/libjansson.so.4 /usr/lib64/libjansson.so
+
+ARG rpm
+
+RUN dnf install -y \
+    --enablerepo=lsst-ts-test \
+    --disablerepo=lsst-ts \
+    MTM1M3TS-22.1.1-10.0.0.el8
```

Comment:
Same as above for m1m3, might be a good idea to fix this before merging to main.